lib/sincerely/services/process_delivery_event.rb

```diff
@@ -47,6 +47,21 @@ def call
 
       attr_reader :event
 
+      def process_bounce_event
+        notification.set_bounced!
+        Sincerely::BounceEvent.create(message_id:, delivery_system:, recipient:, timestamp:)
```

Comment:
I think the `event` object should define the `options` hash that can be saved to the event record
ie (see https://docs.aws.amazon.com/ses/latest/dg/event-publishing-retrieving-sns-examples.html)
```
bounce:
      bounce_type:     event["bounce_type"],
      bounce_subtype:  event["bounce_subtype"],
      action:          event["action"],
      status:          event["status"],
      diagnostic_code: event["diagnostic_code"],



engagement:
      ip_address:      event["ip_address"],
      user_agent:      event["user_agent"],
      link:            event["link"],


compaint:
      user_agent:              event["user_agent"],
      complaint_feedback_type: event["complaint_feedback_type"],

```